app/webpacker/css/darkswarm/_shop-product-rows.scss

```diff
@@ -144,20 +144,16 @@
           }
 
           .product-description {
-            white-space: nowrap;
             overflow: hidden;
             text-overflow: ellipsis;
             margin-bottom: 0.75rem;
             cursor: pointer;
+            
             // Force product description to be on one line
-            // and truncate with ellipsis
+            // and truncate with ellipsis 
```

Comment:
Why did you add this space?
```suggestion
            // and truncate with ellipsis
```

Reply:
It was unintentional...

Reply:
> [filipefurtad0](https://github.com/filipefurtad0) Is this an improvement or does it need more work?

Thanks for the pics @mkllnk , they look good :+1: 

To be sure, I've pulled this branch and tested locally with the [strings which reproduce the issue](https://github.com/openfoodfoundation/openfoodnetwork/issues/10685#issue-1663000868):

Width:
![image](https://user-images.githubusercontent.com/49817236/233168652-34ee7932-aacf-4426-9ffe-e5af559d88de.png)

Length:
![image](https://user-images.githubusercontent.com/49817236/233169087-2851caee-c1c7-4ce1-be38-2ec9597d7b96.png)

So, yes, it looks like it fixes the issue :tada: